parking-lot-help-request/README.md

```diff
@@ -0,0 +1,118 @@
+1. Cut a male jumper wire in half and use a wire stripper to strip at least 1/2" of coating off of both the ends of the jumper wire and each wire connected to the Twidec push button. Also snip off the soldered tip of the button's wires so the filaments are free floating.
+![Twidec button and jumper wire both stripped down to expose their wires for soldering](images/readme-wires-to-solder.jpg)
+_Notice that both the button and jumper wires are exposed for soldering, also there's a length of heat shrink on both button wires that will be used to cover the soldered portion of wire later._
+
+1. Use the "Western Union" X-style of wire wrapping to join the exposed jumper wires together with the Twidec wires and solder them together.
+![Freshly soldered jumper wire and button wire](images/readme-wires-soldered.jpg)
+_A helping hand tool made soldering the jumper wire and button wire together much easier._
+
+1. Optional: Add a small sleeve of heat shrink tubing to each wire before soldering together and after soldering's complete, slide the heat shrink over the exposed wires and use a heat gun to shrink the wrap and protect the wires from the elements. (A great video that shows how do all of this, [wires to heat shrink, is available here](https://www.youtube.com/watch?v=Zu3TYBs65FM&ab_channel=ChrisFix).)
+![Button with heatshrink in place awaiting heatgun application](images/readme-heatgun.jpg)
+_The heat shrink has been shifted over the newly soldered wires - now to shrink it with an application of the heat gun._
+![The button after heatgun application: nice, new jumper wires on each end](images/readme-heatshrink.jpg)
+_The finished button with its newly added jumper wires on each end, and a protective coat of heat shrink to hide the wires._
+
+1. Slide the LED into a breadboard.
+![Sparkfun LED breakout slotted into a standard breadboard](images/readme-led-breadboard.jpg)
+1. Attach the Notecarrier A to the breadboard by plugging a jumper wire from the A's `GND` pin to the `-` bus on the breadboard.
+![Attach a jumper wire from Notecarrier A's GND pin to the breadboard's negative bus](images/readme-notecarrier-breadboard.jpg)
+_Notecarrier A's GND pin to breadboard's negative bus (-)._
+
+1. Attach the Notecarrier A to the LED breakout by plugging jumper wires into its headers in the following configuration:
+   1. `VIO` -> `V5`
+   1. `AUX2` -> `DI`
+   1. `-` -> `GND` (this will be a wire in the negative bus (`-`) of the breaboard to the `GND` pin in the LED)
+![Jumper wires from Notecarrier A connecting it to the LED via jumper wires into the breadboard](images/readme-led-notecarrier.jpg)
+_Notecarrier A VIO pin to V5, AUX2 pin to DI, and negative bus to GND of LED breakout._ 
+
+1.  Attach the newly soldered button wires to the Notecarrier and breadboard:
+    1. Red wire -> `AUX1` on the Notecarrier A
+    2. Black wire -> `-` (negative bus) on the breadboard
```

Comment:
Same comment here on the arrows. There’s actually a unicode arrow you could use too →.

Reply:
➡️